### ISSUE TITLE
Question: Is it possible to use slot with {{> Vue component

### ISSUE THREAD

**Issue by mathieustan** [meteor-vue/blaze-integration#8]
Hello Mitar,

Sorry to bother you. I have a question. 
Is it possible to do something like this : 

```
{{#VueComponent
  component="componentName"
  props=props}}
 Validate // ==> should be a slot in vue
{{/VueComponent}}
```

And in my vue template : 
`<div> <slot>// should render Validate here </slot> <div>`

**Comment by mitar** [comment]
I do not think this would work yet, but I think that in theory it could. But it would be tricky to do it, especially if you want the internal content to be reactive as well. But, yea, this should be supported.

I do not have time at the moment to look into this, so pull requests are encouraged.

**Comment by mathieustan** [comment]
Ok. Thanks for this quick answer. 
I think I should better use <blaze-template> in a vue file. 

If I have more time, i'll try to do it.
